Makefile

```diff
@@ -82,14 +78,8 @@ quad:
 quad-wasm:
 	cd examples && cargo +nightly build --target wasm32-unknown-unknown --features gl --bin quad && wasm-bindgen ../target/wasm32-unknown-unknown/debug/quad.wasm --out-dir ../examples/generated-wasm --web
 
-travis-sdl2:
-	#TODO
-	#if [ -e $(SDL2_CONFIG) ]; then exit 1; fi
-	#mkdir -p $(SDL2_DEST)
-	#test -f $(SDL2_DEST)/dev.deb || curl -sLo $(SDL2_DEST)/dev.deb $(SDL2_PPA)/libsdl2-dev_2.0.3+z4~20140315-8621-1ppa1precise1_amd64.deb
-	#test -f $(SDL2_DEST)/bin.deb || curl -sLo $(SDL2_DEST)/bin.deb $(SDL2_PPA)/libsdl2_2.0.3+z4~20140315-8621-1ppa1precise1_amd64.deb
-	#dpkg-deb -x $(SDL2_DEST)/bin.deb .
-	#dpkg-deb -x $(SDL2_DEST)/dev.deb .
-	#sed -e s,/usr,$(SDL2_DEST),g $(SDL2_CONFIG) > $(SDL2_CONFIG).new
-	#mv $(SDL2_CONFIG).new $(SDL2_CONFIG)
-	#chmod a+x $(SDL2_CONFIG)
+metal-shaders:
```

Comment:
do you think we could rewrite it as `make shader-binaries` that would detect the backends automatically based on the OS?

Reply:
Sure, updated